src/bundler.ts

```diff
@@ -227,10 +226,8 @@ export class Bundler {
    */
   private async _bundleDocument(
       docBundle: AssignedBundle,
-      bundleManifest: BundleManifest,
-      bundleImports?: Set<string>): Promise<ASTNode> {
```

Comment:
Ok.  Adding in separate PR so it doesn't muddy this one. (there are a few other spots including exported functions with unused parameters.)